benchmark/assert/deepequal-buffer.js

```diff
@@ -1,39 +1,63 @@
 'use strict';
+
```

Comment:
Is it preferred to not use a line break after `use strict` in general or would you just want to keep that as a separate concern?